hub2vu/crawling.py

```
@@ -0,0 +1,64 @@
+import requests
+from bs4 import BeautifulSoup
+
+url = "https://namu.wiki/w/%EC%84%9C%EA%B0%95%EB%8C%80%ED%95%99%EA%B5%90"
+
+headers = {
+    "User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 "
+                  "(KHTML, like Gecko) Chrome/118.0.0.0 Safari/537.36"
+}
+response = requests.get(url, headers=headers)
```

Ruff (0.13.1)

Probable use of `requests` call without timeout

(S113)